### PR TITLE
Pylint fixes for build.py

### PR DESCRIPTION
**Pull request by Killswitch00**
**When merged this pull request will:**
- Fix issues found by pylint in tools/build.py

### CODE REVIEW

File: tools/build.py [hunk 1]
```diff
@@ -9,38 +11,48 @@
...
 
 def check_for_obsolete_pbos(addonspath, file):
+    """Return True if the source directory for addon no longer exists"""
     module = file[len(PREFIX):-4]
     if not os.path.exists(os.path.join(addonspath, module)):
         return True
     return False
 
 def main():
+    """main"""
```
Comment (jonpas) [inline tools/build.py:55]: As much as I like standards, Python linters go too far even for my taste. This line is just super redundant.

Comment (Killswitch00) [reply tools/build.py:55]: Yeah, I'll add a pyling ignore for this one

**Review by jonpas — APPROVED** [review]
Looks good, but please don't force push, it is impossible to see what was changed after reviews.

**Comment by Killswitch00** [comment]
My GitHub-foo is a bit rusty. I won't force push again. Thanks for the review!

**Pull request merged**